jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -219,15 +222,107 @@
     suffix:
       - test-unit-integration: # kubernetes-pull-test-unit-integration
           status-context: unit/integration
+          success-comment: ''
+          failure-comment: ''
+          error-comment: ''
+          trigger-phrase: 'unit'
           cmd: |
             export KUBE_VERIFY_GIT_BRANCH="${{ghprbTargetBranch}}"
             ./hack/jenkins/gotest-dockerized.sh
       - verify-all: # kubernetes-pull-verify-all
           status-context: verification
+          success-comment: ''
+          failure-comment: ''
+          error-comment: ''
+          trigger-phrase: 'verify'
           cmd: |
             export KUBE_VERIFY_GIT_BRANCH="${{ghprbTargetBranch}}"
             export KUBE_TEST_SCRIPT="./hack/jenkins/verify-dockerized.sh"
             ./hack/jenkins/gotest-dockerized.sh
+      - build-test-e2e-gce: # kubernetes-pull-build-test-e2e-gce
+          status-context: GCE e2e
+          success-comment: |
+            GCE e2e build/test **passed** for commit ${{ghprbActualCommit}}.
+            * [Build Log](http://pr-test.k8s.io/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/build-log.txt)
+            * [Test Artifacts](https://console.developers.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/artifacts/)
+            * [Internal Jenkins Results](${{JOB_URL}}/${{BUILD_NUMBER}})
+          failure-comment: |
+            GCE e2e build/test **failed** for commit ${{ghprbActualCommit}}.
+            * [Build Log](http://pr-test.k8s.io/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/build-log.txt)
+            * [Test Artifacts](https://console.developers.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/artifacts/)
+            * [Internal Jenkins Results](${{JOB_URL}}/${{BUILD_NUMBER}})
+          error-comment: |
+            GCE e2e build/test **errored** for commit ${{ghprbActualCommit}}.
+            * [Build Log](http://pr-test.k8s.io/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/build-log.txt)
+            * [Test Artifacts](https://console.developers.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/${{ghprbPullId}}/${{JOB_NAME}}/${{BUILD_NUMBER}}/artifacts/)
+            * [Internal Jenkins Results](${{JOB_URL}}/${{BUILD_NUMBER}})
+          trigger-phrase: 'e2e'
+          cmd: |
+            if [[ -d "${{WORKSPACE}}/_output" ]]; then
```

Comment:
are you sure this is needed? `workspace-cleanup` is a wrapper, which means it's run before the build, I think.